src/MyTested.AspNetCore.Mvc.Caching/ComponentBuilderDistributedCacheWithoutExtensions.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace MyTested.AspNetCore.Mvc
+{
+    using System;
+    using System.Collections.Generic;
+    using MyTested.AspNetCore.Mvc.Builders.Base;
+    using MyTested.AspNetCore.Mvc.Builders.Contracts.Base;
+    using MyTested.AspNetCore.Mvc.Builders.Contracts.Data;
+    using MyTested.AspNetCore.Mvc.Builders.Data.DistributedCache.WithoutDistributedCache;
+
+    /// <summary>
+    /// Contains <see cref="Microsoft.Extensions.Caching.Distributed.IDistributedCache"/> extension methods for <see cref="IBaseTestBuilderWithComponentBuilder{TBuilder}"/>.
+    /// </summary>
+    public static class ComponentBuilderDistributedCacheWithoutExtensions
+    {
+        /// <summary>
+        /// Clear all entities from <see cref="Microsoft.Extensions.Caching.Distributed.IDistributedCache"/> service.
+        /// </summary>
+        /// <typeparam name="TBuilder">Class representing ASP.NET Core MVC test builder.</typeparam>
+        /// <param name="builder">Instance of <see cref="IBaseTestBuilderWithComponentBuilder{TBuilder}"/> type.</param>
+        /// <returns>The same component builder.</returns>
+        public static TBuilder WithoutDistributedCache<TBuilder>(
+            this IBaseTestBuilderWithComponentBuilder<TBuilder> builder)
+            where TBuilder : IBaseTestBuilder
+            => builder.WithoutDistributedCache(cache => cache.WithoutAllEntries());
```

Comment:
These chains should be on multiple lines like the example I provided above. Edit the whole file.